src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {

```

Comment:
The `BulldozerProducer` class should parameterize the `MachineProducer` interface with the `Bulldozer` type. According to the task requirements, you should replace `Object` in `get()` with the suitable option, such as `List<Bulldozer> get();`.